nx/lib/nx.ex

```diff
@@ -6851,6 +6855,15 @@ defmodule Nx do
     |> divide(weights_sum)
   end
 
+  defp swap(a, i1, i2) do
+    e1 = Enum.at(a, i1)
+    e2 = Enum.at(a, i2)
```

Comment:
Using the bang version since we expect the index to exist!

```suggestion
    e1 = Enum.fetch!(a, i1)
    e2 = Enum.fetch!(a, i2)
```